doc/examples/transform/plot_ssim.py

```diff
@@ -54,18 +54,16 @@
 ssim_const = ssim(img, img_const,
                   data_range=img_const.max() - img_const.min())
 
-label = 'MSE: {:.2f}, SSIM: {:.2f}'
-
 ax[0].imshow(img, cmap=plt.cm.gray, vmin=0, vmax=1)
-ax[0].set_xlabel(label.format(mse_none, ssim_none))
+ax[0].set_xlabel(f"MSE: {mse_none:.2f}, SSIM: {ssim_none:.2f}")
```

Comment:
This is one case where the original `.format()` usage makes some sense since the same template is reused repeatedly. I am fine with it like this as well, though.